src/components/DatePickerComponent.native.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import DateTimePicker from "@react-native-community/datetimepicker";
+
+interface Props {
+  value: Date;
+  onChange: (e: any, data?: any) => void;
+  mode: "date" | "time" | "datetime";
+  toggleVisibility: () => void;
+}
+const DatePickerComponent: React.FC<Props> = ({
```

Comment:
this'll need to include the withTheme stuff and theme variables so it matches accordingly

Reply:
this is just a wrapper for the component from the library (just to split web and native), it isn't style-able using any colors or styles. The actual DatePicker component is still used and wrapped in a withTheme. Wrapping this withTheme would be of no use I guess.